lib/carbonite/change.ex

```diff
@@ -34,6 +34,7 @@ defmodule Carbonite.Change do
   end
 
   @primary_key false
+  @schema_prefix "carbonite_default"
```

Comment:
testing it for the changes seems bogus, since no one should insert the changes manually, right? Would defeat the purpose of the tool.